site/en/reference/cli/concepts/dataset.md

```diff
@@ -6,6 +6,20 @@ and Pending datasets.
 
 > Key Point: Thread network management can also be done with the [Thread Network SDK for Android](https://developers.home.google.com/thread) and [iOS Thread APIs](https://developer.apple.com/documentation/threadnetwork/).
 
+### WARNING - Restrictions for production use!
+
+The CLI commands to write/change the Active and Pending Operational Datasets may allow setting invalid parameters, or 
```

Comment:
Don't use slashes, instead do "write or change".  Update the other instance below.